packages/liveblocks-yjs/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import LiveblocksProvider from "..";
+
+describe("@liveblocks/yjs", () => {
+  test("TODO", () => {
+    // TODO: Add a real test here
+    expect(LiveblocksProvider).toEqual(LiveblocksProvider);
+  });
+});
```

Comment:
@jrowny FYI, I added in this bogus test in the `liveblocks-yui` workspace, just so that running `npm run test` would not error with a non-zero exit code.